node/service_setup.go

```diff
@@ -42,6 +35,16 @@ func (node *Node) setupForShardLeader() {
 	node.serviceManager.RegisterService(service.Randomness, randomness.New(node.DRand))
 }
 
+func (node *Node) stopShardLeaderServices() {
```

Comment:
These stop funcs seems pretty verbose. Is there a way to loop through the services and and stop signal iteratively?

Reply:
yep. this is better. was thinking about this.